Irony.Tests/050.Irony.Tests.NUnit.csproj

```diff
@@ -1,151 +1,37 @@
-    <Compile Include="OperatorTests.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="NewLineTests.cs" />
-    <Compile Include="RegExLiteralTests.cs" />
-    <Compile Include="StringLiteralTests.cs" />
-    <Compile Include="TestHelper.cs" />
-    <Compile Include="TokenPreviewResolution\ConflictGrammars.cs" />
-    <Compile Include="TokenPreviewResolution\ConflictResolutionTests.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Irony.Interpreter\015.Irony.Interpreter.csproj">
-      <Project>{321A7F5D-00C2-4095-9970-075CDEE8C139}</Project>
-      <Name>015.Irony.Interpreter</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Irony.Samples\020.Irony.Samples.csproj">
-      <Project>{AD263C0B-99D3-40A9-9DBF-9086CC524A0B}</Project>
-      <Name>020.Irony.Samples</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Irony\010.Irony.csproj">
-      <Project>{D81F5C91-D7DB-46E5-BC99-49488FB6814C}</Project>
-      <Name>010.Irony</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="_ Tests - read me.txt" />
-  </ItemGroup>
-  <ItemGroup>
-    <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1 Client Profile</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Windows.Installer.3.1">
-      <Visible>False</Visible>
-      <ProductName>Windows Installer 3.1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\packages\NUnit.3.11.0\build\NUnit.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NUnit.3.11.0\build\NUnit.props'))" />
-  </Target>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net8</TargetFrameworks>

```

Comment:
You might want to target multiple supported frameworks to ensure the tests pass on all of them, e.g. net4x, 6.0, 8.0.

Reply:
nah, I think .net standard is our main target for Irony.dll itself; for dev/test - no need to overcomplicate, .net8 should be on all dev machines now and it is long-term support for the future. I seriously do not expect alternative behavior on 'real' frameworks (net 6 7 8 whatever)

Reply:
> I think .net standard is our main target

Sure. But IMO tests should reflect that. Testing "only" 8.0 while stating it works with any netstandard2.0 compatible fw is risky, even more if testing is free (CI).

Reply:
but then it becomes a real pain to even try to load/run the project. I am so tired of projects with many targets, when to load/browse/run the code you have to install bunch of old crap. Formally, you're right, testing only on .net8 does not prove 100% that it will work on everything, but lets be reasonable. Irony does NOT use any low-level down to hardware/OS features (so far), only very common .net stuff. So if it breaks somewhere, it means that .net version/flavor is 'broken', which is highly unlikely